### ISSUE TITLE
Output of invalid UTF-8 characters when title has extended UTF-8 characters and annotation

### ISSUE THREAD

**Issue by rmonico** [GothenburgBitFactory/taskwarrior#2010]
It happens when I run `task long`, I get following
```
ID Created             Mod   Description                                                                                                                                        
 6 Mon 16/Apr/18 00:48 2min  Another task without special characters
                               16/Apr But with speçiãl characters on annotation
 5 Mon 16/Apr/18 00:48 3min  Task without special chars                                                                                                                         
                               16/Apr Annotation without extended characters                                                                                                    
 4 Mon 16/Apr/18 00:39 11min Seria um arquivo sincronizado em todos os dispositivos com todos os lugares onde tem informações relevantes pra alguma coisa com descri�
                               16/Apr Does it happen just when is there some annotation?
 3 Mon 16/Apr/18 00:37 12min Tarefa com descriç�                                                                                                                               
                               16/Apr Seria um arquivo sincronizado em todos os dispositivos com todos os lugares onde tem informações relevantes pra alguma coisa com descrição
 2 Mon 16/Apr/18 00:37 14min Teste com caracteres extendidos não
 1 Mon 16/Apr/18 00:36 15min Teste                                                                                                                                              

6 tasks
There are local changes.  Sync required.
```

`task show report.long`
```
TASKDATA override: /home/rmonico/.task_bug

Config Variable         Value                                                                                                                
report.long.columns     id,start.active,entry,modified.age,depends,priority,project,tags,recur,wait.remaining,scheduled,due,until,description
report.long.description All details of tasks
report.long.filter      status:pending
report.long.labels      ID,A,Created,Mod,Deps,P,Project,Tags,Recur,Wait,Sched,Due,Until,Description
report.long.sort        modified-
```

My `task diagnostic`:

```
TASKDATA override: /home/rmonico/.task_bug

task 2.5.1
   Platform: Linux

Compiler
    Version: 5.3.0
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      Built: Mar  9 2016 20:12:54
      CMake: 3.4.3
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.4.10
 Build type: 

Configuration
       File: /home/rmonico/.taskrc (found), 1244 bytes, mode 100644
       Data: /home/rmonico/.task_bug (found), dir, mode 40755
   TASKDATA: /home/rmonico/.task_bug
    Locking: Enabled
         GC: Enabled
    $EDITOR: subl3 --new-window --wait
     Server: freecinc.com:53589
         CA: /home/rmonico/.task/freecinc/freecinc_4c3e89d4.ca.pem, readable, 3755 bytes
      Trust: strict
Certificate: /home/rmonico/.task/freecinc/freecinc_4c3e89d4.cert.pem, readable, 3741 bytes
        Key: /home/rmonico/.task/freecinc/freecinc_4c3e89d4.key.pem, readable, 25376 bytes
    Ciphers: NORMAL
      Creds: FreeCinc/freecinc_4c3e89d4/************************************

Hooks
     System: Enabled
   Location: /home/rmonico/.task_bug/hooks
             (-none-)

Tests
      $TERM: xterm-256color (323x65)
       Dups: Scanned 4 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 4 tasks for broken references:
             No broken references found
```

Output of `env`:
```
XDG_SEAT=seat0
XDG_SESSION_ID=c2
GROFF_NO_SGR=1
LC_IDENTIFICATION=pt_BR.UTF-8
_=/usr/bin/env
DISPLAY=:0
BIN=/home/rmonico/bin
SYNCTHING_HOME=/home/rmonico/syncthing
LESS_TERMCAP_ue=
TASKWARRIOR_SCRIPTS_FOLDER=/home/rmonico/.task/scripts
YOUTUBEDL_WRAPPER_FOLDER=/home/rmonico/youtube
COLORTERM=truecolor
TERM=xterm-256color
NLS_LANG=PORTUGUESE.WE8ISO8859P15
WORKON_HOME=/home/rmonico/.virtualenvs
QT_QPA_PLATFORMTHEME=qt5ct
BROWSER=/usr/bin/chromium
LOGNAME=rmonico
LESS_TERMCAP_us=
LC_TIME=pt_BR.UTF-8
SHELL=/bin/zsh
TERMINAL=lxterminal
MUSIC=/home/rmonico/music
LC_NUMERIC=pt_BR.UTF-8
HG=/usr/bin/hg
LS_DEFAULT_PARAMS=-l --group-directories-first --color
LC_PAPER=pt_BR.UTF-8
PATH=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/rmonico/bin:/home/rmonico/bin/stubs:/sbin:/bin:/usr/bin:/usr/local/bin:/usr/local/games:.
GRADLE_HOME=/usr/share/java/gradle
XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session0
PROGRAMS=/home/rmonico/programs
XAUTHORITY=/home/rmonico/.Xauthority
TMSU_HOME=/home/rmonico/tmsu
GTK2_RC_FILES=/home/rmonico/.gtkrc-2.0
_JAVA_OPTIONS=-Dawt.useSystemAAFontSettings=on -Dswing.aatext=truedd
XDG_SESSION_DESKTOP=i3
LC_MEASUREMENT=pt_BR.UTF-8
LC_ADDRESS=pt_BR.UTF-8
MOZ_PLUGIN_PATH=/usr/lib/mozilla/plugins
GDMSESSION=i3
MAVEN_OPTS=-Xmx512m
PIP_DOWNLOAD_CACHE=/home/rmonico/.pip_download_cache
TNS_ADMIN=/home/rmonico/.config/oracle
LESS=-iSR
MAIL=/var/spool/mail/rmonico
DESKTOP_SESSION=i3
LESS_TERMCAP_se=
EVENT_FOLDER=/home/rmonico/syncthing/eventos
XDG_RUNTIME_DIR=/run/user/1000
DEV_LIBRARY_PATH=/home/rmonico/repos/cruzeiro/migrado/__library__
LESS_TERMCAP_so=
GTK_MODULES=canberra-gtk-module
PWD=/home/rmonico
YOUTUBEDL_PARAMS_FOR_AUDIO=--format worstvideo+bestaudio --extract-audio
VTE_VERSION=5201
WGET_DEFAULT_PARAMS=--continue --timeout=10 --tries=inf --waitretry=10
LC_MONETARY=pt_BR.UTF-8
MACRO_HOME=/home/rmonico/macros
USER=rmonico
HOME=/home/rmonico
PWMENU_HOME=/home/rmonico/syncthing/data/config/pwmenus
XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
locker=xscreensaver-command -lock
XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/rmonico
TASKWARRIOR_ALT_DATEFORMAT=d/bTH:N
LANG=pt_BR.UTF-8
LC_NAME=pt_BR.UTF-8
SHLVL=2
XDG_VTNR=7
GDM_LANG=pt_BR.utf8
EDITOR=subl3 --new-window --wait
XDG_SESSION_TYPE=x11
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
LESS_TERMCAP_mb=
XDG_CURRENT_DESKTOP=i3
LESS_TERMCAP_md=
LC_TELEPHONE=pt_BR.UTF-8
LESS_TERMCAP_me=
OLDPWD=/home/rmonico
ZSH=/home/rmonico/.oh-my-zsh
PAGER=less
LC_CTYPE=pt_BR.UTF-8
LSCOLORS=Gxfxcxdxbxegedabagacad
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
VIRTUALENVWRAPPER_PROJECT_FILENAME=.project
VIRTUALENVWRAPPER_WORKON_CD=1
VIRTUALENVWRAPPER_SCRIPT=/usr/bin/virtualenvwrapper.sh
VIRTUALENVWRAPPER_HOOK_DIR=/home/rmonico/.virtualenvs
TASKDATA=/home/rmonico/.task_bug
```

My .task_bug folder:
[task_bug.tar.gz](https://github.com/GothenburgBitFactory/taskwarrior/files/1913828/task_bug.tar.gz)

If is there something else I can provide to help to solve this issue I'm at disposal. Thanks in advance!

**Comment by rmonico** [comment]
I think this were fixed in 2.6.0, but I will make more tests before close this.

**Comment by pbeckingham** [comment]
This is already resolved, but unreleased.
